kanshi.5.scd

```diff
@@ -35,6 +35,37 @@ Directives are followed by space-separated arguments. Arguments can be quoted
 	On *sway*(1), output names and descriptions can be obtained via
 	*swaymsg -t get_outputs*.
 
+*exec* <command>
+	An exec directive executes a command when the profile was successfully
+	applied. This can be used to update the compositor state to the profile
+	when not done automatically.
+
+	Commands are executed asynchronously and their order may not be preserved.
+	If you need to execute sequential commands, you should collect in one exec
+	statement or in a separate script.
+
+	On *sway*(1) for example, *exec* can be used to move workspaces to the
+	right output:
+
+```
+	multihead {
+		output eDP-1 enable
+		output DP-1 enable transform 270
+		exec swaymsg workspace 1, move workspace to eDP-1
+	}
+```
+
+	Note that some extra care must be taken with outputs identified by an
+	output description as the real name may change:
+
+```
+	complex {
+		output "Hewlett Packard LA2405 CZ40022966" mode 1920x1200
+		exec swaymsg workspace 1, move workspace to output $(swaymsg -p -t get_outputs | grep "Hewlett Packard LA2405 CZ40022966" | cut -f2 -d" ")
```

Comment:
`move workspace to output "Hewlett Packard LA2405 CZ40022966"` should just work

Reply:
Oh, yeah, i3 legacy…

Reply:
no, it does not. But testing further shows that `move workspace to output '"Hewlett Packard LA2405 CZ40022966"'` does. Changing to that, but it is potentially a bug in sway that we want to solve. Not sure that adding it here is a good idea.

Reply:
Resolved then ;-)